### PR TITLE
Allow x-mode interaction to return only 1 item.

### PR DESCRIPTION
**Pull request by psaulic**
This additional option allows for a simpler implementation of a vertical cursor, returning only 1 item per dataset even for densely populated graphs. Without this option, densely packed graphs return multiple points, cluttering the tooltip and making it unclear which value is being displayed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
   },
   "dependencies": {
     "chartjs-color": "^2.1.0",
+    "gulp-uglifyes": "0.1.2",
```
Comment (minjaitk) [inline package.json]: not sure how I feel about adding a new package for syntax purposes and 8 new lines of code

File: gulpfile.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ var insert = require('gulp-insert');
 var replace = require('gulp-replace');
 var size = require('gulp-size');
 var streamify = require('gulp-streamify');
-var uglify = require('gulp-uglify');
+var uglify = require('gulp-uglifyes');
```
Comment (minjaitk) [inline gulpfile.js]: I don't like the idea of replacing an existing package just so you can use a certain syntax.
you cannot be sure whether you broke something else with this change.

**Review by minjaitk — CHANGES_REQUESTED** [review]
do this without package changes

**Comment by psaulic** [comment]
@minjaitk Please review the latest changes.

**Review by minjaitk — APPROVED** [review]

**Pull request merged**